docs/src/pages/[platform]/getting-started/troubleshooting/troubleshooting.react.mdx

```diff
@@ -189,7 +195,9 @@ export default defineConfig({
 
 ## Create React App
 
-When using [Geo components](/connected-components/geo) and Create React App v5, users may experience the following error when rendering the `<MapView>` component in a production build:
+### CRA 4:`Uncaught ReferenceError: g is not defined`
+
+When using [Geo components](/connected-components/geo) and Create React App v4, users may experience the following error when rendering the `<MapView>` component in a production build:
```

Comment:
The original text was referencing CRA 5, is this change intentional?

Reply:
Good catch. Yes. I was about to comment that. 

It fixed a typo. Should be CRA4 https://github.com/aws-amplify/amplify-ui/issues/1656

Reply:
Thanks for making these changes! Love that the table of contents makes all the troubleshooting items visible for better discoverability!

Reply:
Kudos to @zchenwei for telling me that 🙌